CMakeLists.txt

```diff
@@ -386,8 +386,8 @@ if(ENABLE_CPACK)
       "${CPACK_RPM_PACKAGE_REQUIRES},OCE-draw,OCE-foundation,OCE-modeling,OCE-ocaf,OCE-visualization,"
       "poco-crypto,poco-data,poco-mysql,poco-sqlite,poco-odbc,poco-util,poco-xml,poco-zip,poco-net,poco-netssl,poco-foundation,"
       "sip >= 4.18,python-enum34,"
-      "python-six,python-ipython >= 1.1.0,python-ipython-notebook,PyYAML,"
-      "python-requests,"
+      "python-six,python-ipython >= 1.1.0,python-ipython-notebook,"
+      "python-requests, python-toml, PyYAML"
```

Comment:
I think we are missing a comma at the end of this line and that's why the system tests are failing.

Could you also remove the whitespace after the comma for consistency?

Reply:
Thanks for spotting that